viz_scripts/timeseries_analysis.ipynb

```diff
@@ -49,82 +49,38 @@
     "import emission.storage.decorations.timeline as esdl\n",
     "import emission.storage.timeseries.abstract_timeseries as esta\n",
     "import emission.storage.timeseries.timequery as estt\n",
+    "from plots import *\n",
+    "import scaffolding\n",
```

Comment:
Instead of adding these imports to the other notebooks, you should move the high level plots from here into plots. You might even want to create a separate `timeseries_plots` that is separate from `plots` since the graphs here are so different.

Reply:
Added wrapper function for each timeseries plot to plots.py